scaleway/table_scaleway_account_project.go

```diff
@@ -0,0 +1,133 @@
+package scaleway
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/scaleway/scaleway-sdk-go/api/account/v3"
+	"github.com/scaleway/scaleway-sdk-go/scw"
+
+	"github.com/turbot/steampipe-plugin-sdk/v5/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin/transform"
+)
+
+//// TABLE DEFINITION
+
+func tableScalewayAccountProject(_ context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "scaleway_account_project",
+		Description: "A Scaleway Account Project.",
+		List: &plugin.ListConfig{
+			Hydrate: listAccountProjects,
+		},
+		Columns: []*plugin.Column{
+			{
+				Name:        "project_id",
+				Description: "The ID of the project.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("ID"),
+			},
+			{
+				Name:        "name",
+				Description: "Name of the project.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "created_at",
+				Description: "The time when the project was created.",
+				Type:        proto.ColumnType_TIMESTAMP,
+			},
+			{
+				Name:        "updated_at",
+				Description: "The time when the project was updated.",
+				Type:        proto.ColumnType_TIMESTAMP,
+			},
+			{
+				Name:        "description",
+				Description: "Description of the project.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "organization",
```

Comment:
```suggestion
				Name:        "organization_id",
```

I think adding `_id` would be helpful so it's clear what this column contains specifically.

As part of this PR, can you please also update the column name in the `scaleway_iam_user` table so they're consistent (and any other places I missed that have this column)?

Reply:
I changed by organization_id